src/sentry/auth/superuser.py

```diff
@@ -39,10 +41,10 @@
 COOKIE_HTTPONLY = getattr(settings, "SUPERUSER_COOKIE_HTTPONLY", True)
 
 # the maximum time the session can stay alive
-MAX_AGE = getattr(settings, "SUPERUSER_MAX_AGE", timedelta(hours=4))
+MAX_AGE = getattr(settings, "SUPERUSER_MAX_AGE", timedelta(minutes=15))
```

Comment:
this means a superuser session can only last 15 minutes, no?

Reply:
Yup, we want to limit the superuser session to 15 mins

Reply:
whats driving this? its going to make _admin really hard to use

Reply:
https://www.notion.so/sentry/23Q1-Better-accountability-for-superuser-access-3a9d48bc2326467990c70f771bfa32a0

Reply:
that makes sense, and the premise of this makes sense, but i think theres a large delta in accessing customer data (e.g. viewing an orgs data) and utilizing _admin which is accessing sentry data (we have definitions for this buried in notion). if there were a way to say "you can access customer's data aka events for 15 minutes when you put in a reason" and it kept the 4 hour timer on _admin/general management stuff i think that'd be a great win

however with this I see something like: im in _admin, i click around, im 12 minutes into my session and uploading a file for a policy change, and the session has expired.

Reply:
hmm good point, ill bring it up for discussion

Reply:
after discussion, we plan to keep it at 4 hours unless you're accessing another org, in which then it will be 15 mins